Automattic/pocket-casts-android
# Save trending podcasts in the database

## Description

We need to save trending podcasts in the database for the top secret project to display them in the UI.

Internal ref: pdeCcb-520-p2

## Testing Instructions

1. Open the app.
2. Go to trending.
3. Query `SELECT * FROM trending_podcasts` in the App Inspection tool.
4. You should see some podcasts.

## Checklist
 
#### I have tested any UI changes...
